### DISCUSSION TITLE
para facturacion argentina y demas utilidades puedo instalar enterprise ?

### DISCUSSION THREAD

**Discussion by 3lk0k0** [OdooAR/odoo-argentina-doc#12] [General]
desconozco si puedo isntalar enterprise o el comunity para usar la facturacion argentina y demas cosas

**Comment by joseluisbossio** [comment]
Si

**Comment by ibuioli** [comment]
Enterprise tiene su propia localización que viene con el pago de la licencia. En community hay varias localizaciones para facturación, históricamente está la de AdHoc para 13 y ahora para 15; la de Moldeo para 13 y 14, Codize está manteniendo una para 14 y proximamente 15; QuilSoft también tiene una. Ahi depende del gusto y de las funcionalidades que te interesen, pero en ambas ediciones tenés facturación.
